pkg/api/openshiftcluster.go

```diff
@@ -255,6 +264,7 @@ type NetworkProfile struct {
 	ServiceCIDR            string                 `json:"serviceCidr,omitempty"`
 	SoftwareDefinedNetwork SoftwareDefinedNetwork `json:"softwareDefinedNetwork,omitempty"`
 	MTUSize                MTUSize                `json:"mtuSize,omitempty"`
+	OutboundType           OutboundType           `json:"outboundType,omitempty" mutable:"true"`
```

Comment:
any idea why this is marked `mutable:"true"` here? If not, likely worth removing to sync with https://github.com/Azure/ARO-RP/pull/2248/files#diff-f9a5398ee035bbae6c18fc8468081bbbd9bab4d81504809a5589541ca2171aa7R267

Reply:
Dang, looks like I grabbed that from the wrong spot, I'll fix it up.